Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -221,6 +268,14 @@ class AuthenticationSettingsViewController: SettingsTableViewController {
             navigationItem.title = AuthenticationStrings.passcode
         }
     }
+    
+    private var authenticationSectionTitle: NSAttributedString {
+        if LAContext().canEvaluatePolicy(.DeviceOwnerAuthenticationWithBiometrics, error: nil) {
+            return NSAttributedString(string: AuthenticationStrings.authenticateFeaturesWithTouchIDPasscode)
+        } else {
+            return NSAttributedString(string: AuthenticationStrings.authenticateFeaturesWithPasscode)
+        }
```

Comment:
More of a design question but would it technically make more sense to have the title just be `Prompt for Authentication to access:` instead of `Use Touch ID & Passcode to access/Use Passcode to access`